### DISCUSSION TITLE
Disk full on forum server

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/CrispCMS#56] [General]
We had a disk full problem over the weekend, on forum.tosdr.org.

@JustinBack I'm opening this issue because you reported that https://forum.tosdr.org is still displaying disk-full errors.

But it seems to be only 65% full at the moment (8.6 out of 25 Gb available). Is there still a problem?

**Comment by JustinBack** [comment]
Not anymore no, I have pruned docker images which were piling up to 7 Gigabytes and they seem to not regenerate. We require a minimum of 5 GB Free disk space to update and run discourse
